Cloud Application Manager/Deploying Anywhere/Using-CenturyLink-Private-Cloud-with-VMC-on-AW

```diff
@@ -0,0 +1,134 @@
+| Tasks | <li>Create tasks</li> |
+| Virtual machine (Configuration) | <li>Add new disk</li><li>Add or remove device</li><li>Advanced</li><li>Change CPU count</li><li>Change resource</li><li>Memory</li><li>Modify device settings</li><li>Remove disk</li><li>Rename</li><li>Reset guest information</li><li>Set annotation</li><li>Settings</li> |
+| Virtual machine (Interaction) | <li>Answer question</li><li>Configure CD media</li><li>Power Off</li><li>Power On</li> |
+| Virtual machine (Inventory) | <li>Create from existing</li><li>Create new</li><li>Remove</li> |
+| Virtual machine (Provisioning) | <li>Customize</li><li>Deploy template</li><li>Modify customization specification</li><li>Read customization specifications</li> |
+
+### Bootstrapping virtual machine templates with the Cloud Application Manager Agent
+
+VMC templates need elasticbox-init to allow the Cloud Application Manager agent to execute box scripts at deploy time.
+
+**Linux**
+
+Follow these steps to install elasticbox-init on a Linux template.
+
+
+1. Log in to the VMC client and open the Linux virtual machine.
+
+2. SSH into the virtual machine.
+
+3. [Install the VMware tools.](https://www.vmware.com/support/ws55/doc/ws_newguest_tools_linux.html)
+
+4. Run this command with root privileges to install elasticbox-init:
+
+   ```
+	 curl -L https://cam.ctl.io/agent/linux/vsphere/template_customization_script.sh | sudo bash
+	 ```
+
+   * **Note:** If running Cloud Application Manager as an appliance, replace cam.ctl.io with the appliance hostname or IP address.
+
+**Windows**
+
+Follow these steps to run a script that creates a scheduled task on a Windows Server 2012 template. When you deploy, Cloud Application Manager clones this template and installs the agent using the scheduled task.
+
+1. Log in to the Windows Server 2012 virtual machine template using remote desktop protocol (RDP).
+
+2. [Install the VMware tools](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=1018377).
+
+3. [Download the scheduled task script.](https://cam.ctl.io/agent/windows/vsphere/template_customization_script.ps1)
+
+   * **Note:** If running Cloud Application Manager as an appliance, replace cam.ctl.io with the appliance hostname or IP address.
+
+4. Right-click the script and click **Run PowerShell**.
+
+### Registering a CenturyLink Private Cloud with VMC on AWS provider in Cloud Application Manager
+
+In order to deploy to a VMC on AWS, you must first provide information to connect. 
+
+**Steps**
+
+1. In Cloud Application Manager, click **Providers** > **New Provider**.
+
+2. Select **CenturyLink Private Cloud with VMC on AWS**.
+
+3. Enter the endpoint URL for the VMC server and a username, password to the VMC API.
+   * **Note:** The endpoint URL can be a hostname or an IP address, and has to be prefixed with **https://**. Be sure to use https and not http.
+
+   ![Creating a new VMC provider](../../images/cloud-application-manager/VMC01.png)
+
```

Comment:
Please, include also a section to deploy new instances, and link to the vCenter provider section for it.